src/devices/cpu/sh/sh7021.h

```diff
@@ -0,0 +1,54 @@
+// license:BSD-3-Clause
+// copyright-holders:Juergen Buchmueller, R. Belmont, Angelo Salese, ...
+
+#ifndef MAME_CPU_SH_SH7021_H
+#define MAME_CPU_SH_SH7021_H
+
+#pragma once
+
+#include "sh2.h"
+
+class sh2a_sh7021_device : public sh2_device
+{
+public:
+	// construction/destruction
+	sh2a_sh7021_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+protected:
+	// device-level overrides
+	virtual void device_start() override;
+	virtual void device_reset() override;
```

Comment:
Can we please lose this copy/pasted comment?  If you want to label the `device_t` implementation, make the comment read “// device_t implementation”.